cordapp-cpk2/README.md

```diff
@@ -306,7 +303,7 @@ to ensure that the bundle respects OSGi's [Service Loader Mediator Specification
 
 ## Corda Metadata
 
-The plugin will generate the following tags in the "main" jar's `MANIFEST.MF` by default:
+The plugin will generate the following tags in the jar's `MANIFEST.MF` by default:
 - `Corda-Contract-Classes`
 - `Corda-Flow-Classes`
 - `Corda-MappedSchema-Classes`
```

Comment:
I think this section is no longer true, and should be removed. All Corda Metadata tags are now imported from the `cordapp-configuration` Gradle plugin.

Reply:
I've deleted the list of tags and changed the paragraph to:

> The plugin will generate `Corda-*-Classes` tags in the jar's `MANIFEST.MF`. The generated tags are controlled by the `cordapp-configuration` Gradle plugin.

Let me know if you wanted me to delete the whole "Corda Metadata" section.

Reply:
No harm in keeping it, I guess. But in that case, we should call the plugin by its full Gradle plugin ID `net.corda.cordapp.cordapp-configuration`, and explain where to get it from.

Reply:
Done